app/src/main/java/com/cp/tasky/auth/shared/data/Constant.kt

```diff
@@ -0,0 +1,10 @@
+package com.cp.tasky.auth.shared.data
+
+object Constant {
```

Comment:
I'd advise against having one global Constants file since it could literally contain anything. For small projects this might be more readable since you know where to look for a constant, but this approach doesn't scale (imagine you'd have 200 constants and want to find one).
Instead, I would recommend separating them by context. All these constants here seem to be related to session storage, so it could make sense including them in a companion object of your session storage implementation.